### PR TITLE
Corrected sha256 sum for jq sources

### PR DESCRIPTION
**Pull request by svetlyak40wt**
Seems, that during upgrade somebody forgot about it.

Now:

```
[art@osx:~]% curl -s -L 'https://github.com/stedolan/jq/releases/download/jq-1.6/jq-1.6.tar.gz' | shasum -a 256
5de8c8e29aaa3fb9cc6b47bb27299f271354ebb72514e3accadc7d38b5bbaa72  -
```

Without it, I get checksum mismatch when trying to upgrade jq on El Captain OSX.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fxcoudert** [comment]
It was updated: https://github.com/Homebrew/homebrew-core/commit/c3be568fc3c621bebae607b2352e590e86353742 and it was correct at the time (otherwise our test bot would have shown an error).

Can you ask upstream why the checksum has changed? We need to confirm that it is not a malicious change of code before we update in Homebrew.

**Comment by svetlyak40wt** [comment]
Created an issue in the upstream: https://github.com/stedolan/jq/issues/1773

**Comment by SMillerDev** [comment]
Might be related to https://github.com/stedolan/jq/issues/1352 but let's wait for an official answer from upstream

**Comment by marques-work** [comment]
@fxcoudert upstream confirmed the SHA change. Can this be merged now?

See https://github.com/stedolan/jq/issues/1773#issuecomment-441624322

**Review by fxcoudert — APPROVED** [review]

**Comment by igas** [comment]
Hi @svetlyak40wt 

It failed with:

```
sha256 changed without the version also changing; please create an issue upstream to rule out malicious circumstances and to find out why the file changed.
```

I guess maintainers will merge it anyway as you confirmed it in the upstream.

You can check it by clicking **Details**
image

And later to the test itself:

image

**Comment by fxcoudert** [comment]
Thanks @svetlyak40wt

**Pull request closed without merging**
